src/components/FlagStatusBadge/FlagStatusBadge.tsx

```
@@ -0,0 +1,29 @@
+    enabled ? styles["badge-enabled"] : styles["badge-disabled"],
+  );
+
+  const indicatorClass = mergeClasses(
+    styles.indicator,
+    enabled ? styles["indicator-enabled"] : styles["indicator-disabled"],
+  );
+
+  return (
+    <span
+      className={badgeClass}
+      role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)